2-quickie-application/src/test/java/org/pinkcrazyunicorn/quickie/application/profile/ProfileServiceTest.java

```diff
@@ -124,7 +124,7 @@ public void testProfileSetOpinionAbout() {
     }
 
     @Test
-    public void testSetOpinionAboutUnavailable() {
+    public void ShouldThrowWhenSettingOpinionForInvalidProfile() {
```

Comment:
Bitte Lower CamelCase ^^ Und "should throw" finde ich nicht so schön. Vllt besser "should err".